spec/unit/build_task_spec.rb

```diff
@@ -144,6 +144,15 @@
         build_task.build(system_description, output_path)
       }.to raise_error(Machinery::Errors::BuildFailed, /i586/)
     end
+
+    it "shows an error when the current user does not have access to the image directory path" do
+      allow(LocalSystem).to receive(:validate_architecture)
+      allow_any_instance_of(Os).to receive(:architecture).and_return("x86_64")
+
+      expect {
+        build_task.build(system_description, "/test")
+      }.to raise_error(Machinery::Errors::BuildDirectoryCreateError)
```

Comment:
It might make sense to verify that the path and the username are put out correctly.